### PR TITLE
Test duration set in one place

### PR DESCRIPTION
**Pull request by aleksbykov**
Change stress* methods to use  get_duration method to set timeout

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -724,9 +724,7 @@ def run_stress(self, stress_cmd, duration=None):
     def run_stress_thread(self, stress_cmd, duration=None, stress_num=1, keyspace_num=1, profile=None, prefix='',
                           keyspace_name='', round_robin=False, stats_aggregate_cmds=True):
         # stress_cmd = self._cs_add_node_flag(stress_cmd)
-        if duration is None:
-            duration = self.params.get('test_duration')
-        timeout = duration * 60 + 600
+        timeout = self.get_duration(duration)
```
Comment (bentsi) [inline sdcm/tester.py]: - make it as @property
- call it test_duration

Comment (aleksbykov) [reply sdcm/tester.py]: If set as property, then either we need add setter, to process stress_* method param (duration) and recalculate it in seconds or we need to store the 2 durations: one from configuration and one from method's param.

Comment (bentsi) [reply sdcm/tester.py]: please do not mark resolved on something that is in discussion

Comment (bentsi) [reply sdcm/tester.py]: why do we need setter?

Comment (bentsi) [reply sdcm/tester.py]: by making it property you will remove a lot of redundant code. Don't think about recalculation, we are not Real time process running on embedded chip :)

Comment (aleksbykov) [reply sdcm/tester.py]: i mean next: 
 def run_stress_thread(self, stress_cmd, duration=None,
could take a parameter duration from test itself, and not from configuration. and them passed to stress in seconds. if set as property, then i need return the conditions to each command to check was duration parameter passed, and recalculate it to seconds.(this is as it was before).

Using the function allow to encapsulate this logic in on on place: getting the duration from config or check passed duration, and calculate the seconds for any variant.

Comment (bentsi) [reply sdcm/tester.py]: Looked in the code and didn't find any call to `run_stress_thread` where duration is explicitly passed.
Currently `test_duration` is meant to be used as timeout to the stress command, so I don't want to change the funcitonality, but in future test_duration should be equal to test_timeout. So please implement it as property decorator and remove all the mentions in `run_stress_thread` of the duration

Comment (aleksbykov) [reply sdcm/tester.py]: Found passing duration in next tests:
- grow_cluster_test.py: on string 88
- maintenance_test.py: on string 35
- mgmt_cli_test.py: on string 193
- reduce_cluster_test.py: on string 98
- upgrade_schema_test.py: on string 293

**Pull request merged**